### ISSUE TITLE
Error inserting german symbols

### ISSUE THREAD

**Issue by Eisfreak7** [lervag/vimtex#267]
Whenever I'm writing German symbols (ä, ü, ß), vim shows one of two errors at the bottom:
- ```IndexError: bitset::set: __posotion (which is 18446744073709551555) >= _NB (which is 128)```
- ```IndexError: string index out of range```

Which one of the errors appers seems to be random. The error clears when I exist insert mode and it doesnt seem to have any inpact on text editing. It only shows when I edit tex files. It shows in both vim and neovim.

**Comment by lervag** [comment]
This is strange, and I suspect it is not due to vimtex. Could you please provide a minimal working example on how to reproduce this? A minimal vimrc file that loads vimtex and a step by step procedure for how to get the error. See [here](https://github.com/lervag/dotvim/blob/master/minivimrc) for an example of a minimal vimrc file.

**Comment by Eisfreak7** [comment]
Indeed, its not vimtex. I could narrow it down to YouCompleteMe and then found [this issue](https://github.com/Valloric/YouCompleteMe/issues/1378) which seems to describe the problem.
I should of course have tested that before. But since I observed this problem only when editing tex files (presumably because these are generally the only files I write german in) and it was the newest addons I installed, I just assumed the problem was with vimtex.

Sorry for the false issues and thank you for your time!
